Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM maartje/dind-golang:docker-18.09.7-go-1.12.7
```

Comment:
Add a TODO to change this as we might forget why you use my image ;)

Reply:
Done